packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -5627,6 +6107,25 @@ The catalog is currently not exposed via an API. While additional apps may be ad
 
 The current workaround is to manually configure the desired application via the administrator UI in a preview (sandbox) organization and view the application via [Get Application](#get-application).
 
+##### App settings
+
+Each application has a schema that defines the required and optional settings for the application. When adding an application, you must specify the required settings.
+
+The catalog is currently not exposed via an API. While additional apps may be added via the API, only the following template applications are documented:
+
+The current workaround is to manually configure the desired application via the administrator UI in a preview (sandbox) organization and view the application via [Get Application](#get-application).
```

Comment:
The current solution is to manually configure the desired application using the Okta Admin Dashboard and a preview (sandbox) Okta org. You can then view the application details using the [Get Application](#get-application) API.